### ISSUE TITLE
crash w/ binflop upload

### ISSUE THREAD

**Issue by Pemigrade** [Pemigrade/botflop#33]
```
[20:41:43 [31merror[39m]: unhandledRejection: Cannot read property 'split' of null
TypeError: Cannot read property 'split' of null
    at module.exports (/home/container/events/messageCreate.js:28:60)
    at Client.emit (node:events:394:28)
    at MessageCreateAction.handle (/home/container/node_modules/discord.js/src/client/actions/MessageCreate.js:26:14)
    at Object.module.exports [as MESSAGE_CREATE] (/home/container/node_modules/discord.js/src/client/websocket/handlers/MESSAGE_CREATE.js:4:32)
    at WebSocketManager.handlePacket (/home/container/node_modules/discord.js/src/client/websocket/WebSocketManager.js:346:31)
    at WebSocketShard.onPacket (/home/container/node_modules/discord.js/src/client/websocket/WebSocketShard.js:478:22)
    at WebSocketShard.onMessage (/home/container/node_modules/discord.js/src/client/websocket/WebSocketShard.js:317:10)
    at WebSocket.onMessage (/home/container/node_modules/ws/lib/event-target.js:199:18)
    at WebSocket.emit (node:events:394:28)
    at Receiver.receiverOnMessage (/home/container/node_modules/ws/lib/websocket.js:1169:20)
```

**Comment by saboooor** [comment]
why is contentType nullable burh

**Comment by saboooor** [comment]
fixed in #34
